### DISCUSSION TITLE
Bug in function navego_plot_main

### DISCUSSION THREAD

**Discussion by rodralez** [rodralez/NaveGo#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shjdcj** [comment]
Line 213 in function navego_plot_main may have the mistake
![图片](https://user-images.githubusercontent.com/34773765/149613946-b7da07e6-cfb0-465e-bf16-ff318ffc2180.png)

**Reply by rodralez** [reply]
Hello, I checked both branches, master and develop, and I cannot find this copy&paste bug.

**Comment by shjdcj** [comment]
Hi 
it's in the master branch,  NaveGo/performance-analysis/navego_plot_main.m , Line 213

**Reply by rodralez** [reply]
Thank you very much for pointing out this error!

The problem was that my local master repository was not synced with the github master repository. I already solved the problem.

Kind regards.
